faq_cn.html

```diff
@@ -0,0 +1,474 @@
+                        </div>
+                        <div class='grid-element'>
+                            <img title="Cobblestone" class="displayed" src='img/items/cobblestone.png' />
+                        </div>
+                        <div class='grid-element'>
+                            <img title="Sand" class="displayed" src='img/items/sand.png' />
+                        </div>
+                        <div class='grid-element'>
+                            <img title="Sand" class="displayed" src='img/items/sand.png' />
+                        </div>
+                        <div class='grid-element'>
+                            <img title="Sand" class="displayed" src='img/items/sand.png' />
+                        </div>
+                        <div class='grid-element'>
+                            <img title="Sand" class="displayed" src='img/items/sand.png' />
+                        </div>
+                    </div>
+                </div>
+                <div class="recipe-boxes">
+                    <div class='grid'>
+                        <h5>烈焰棒</h5>
+                        <div class='grid-element'>
+                            <img title="Glowstone Dust" class="displayed" src='img/items/glowstone_dust.png' />
+                        </div>
+                        <div class='grid-element'>
+                            <img title="White Stained Glass" class="displayed" src='img/items/white_stained_glass.png' />
+                        </div>
+                        <div class='grid-element'>
+                            <img title="Glowstone Dust" class="displayed" src='img/items/glowstone_dust.png' />
+                        </div>
+                        <div class='grid-element'>
+                            <img title="Glowstone Dust" class="displayed" src='img/items/glowstone_dust.png' />
+                        </div>
+                        <div class='grid-element'>
+                            <img title="White Stained Glass" class="displayed" src='img/items/white_stained_glass.png' />
+                        </div>
+                        <div class='grid-element'>
+                            <img title="Glowstone Dust" class="displayed" src='img/items/glowstone_dust.png' />
+                        </div>
+                        <div class='grid-element'>
+                            <img title="Glowstone Dust" class="displayed" src='img/items/glowstone_dust.png' />
+                        </div>
+                        <div class='grid-element'>
+                            <img title="White Stained Glass" class="displayed" src='img/items/white_stained_glass.png' />
+                        </div>
+                        <div class='grid-element'>
+                            <img title="Glowstone Dust" class="displayed" src='img/items/glowstone_dust.png' />
+                        </div>
+                    </div>
+                </div>

```

Comment:
The changes to the English FAQ page haven't been merged into the `cn` branch yet, but it would be nice to add the recipes for lily pads and vines (see ecbd0df74fb0915b2a91e170372a6d6c44f30e4e)

Reply:
@SnUncle